src/components/app/profile/profile-search/profile-search.js

```diff
@@ -9,13 +9,33 @@ export default Component ({
 
 class ProfileSearch extends WebComponent {
 
+    redirectToSearchPage() {
+        const searchTerm = this.shadowRoot.querySelector('#input-control').value.trim();
+        if (searchTerm) {
+            window.location.href = `/app/search/${encodeURIComponent(searchTerm)}`;
```

Comment:
```js
Router.push(`/app/search/${encodeURIComponent(searchTerm)}`)
```
 Ejemplo en src/pages/sidebar/MainSidebar.js línea 104

Reply:
By the way, la función encodeURIComponent no existen no?

Reply:
si existe
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent

Reply:
lo del sidebar lo cmabio ahora. Sabia que algo habia, pero no me acordaba cual concretamente xd

Reply:
Ah mira no sabía que existía, muy util 👯